ember-flight-icons/README.md

```diff
@@ -37,13 +37,13 @@ The `name` must be specified. A default icon size of 24 will be provided.
 
 ### Optional usage
 
-Icons are set to a `viewBox` square size of 24 by default. 
+Icons are set to a `viewBox` square size of 24 by default.
```

Comment:
(not blocking) i think we have different editorconfig's here. i've used markdownlint and am pretty happy with it, if we want to go that route

Reply:
I have markdownlint as well. Happy to sync settings, I just use the default settings.

Reply:
https://github.com/hashicorp/flight/issues/92